tests/integration/pybaseball/test_team_game_logs.py

```diff
@@ -0,0 +1,13 @@
+import pybaseball
+
+
+class TestTeamGameLogs:
```

Comment:
Class wrapper shouldn't be here for integration testing. Once you remove the class thingy it should work just fine